.storybook/preview-head.html

```diff
@@ -399,7 +398,7 @@
 		border: 1px solid #333333;
 	}
 
-	.tokens__list__dropdown {
+	.tokens__list__item {
```

Comment:
i think this may have broken the styling of the deprecated items? 

<img width="314" alt="image" src="https://user-images.githubusercontent.com/867233/180180569-5b921604-5f65-4b35-8314-f467a10a0d62.png">

Reply:
I think the styling is broken because by default storybook applies it's own styles (font / spacing) to `.mdx` files. We may have to add our own to get it looking like the other stories.

Reply:
Resolved this by adding the text styles manually.